### PR TITLE
V222 redirect

### PR DESCRIPTION
**Pull request by rstoneback**
# Description

Addresses #617 

Folds in some documentation changes from the v2.2.2 release. Plus includes a fix for the zenodo.json.

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

I don't currently know how to test the .json without a release.

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -1,58 +1,56 @@
...
         "orcid": "0000-0001-8340-5625"
-      }, 
+      },
+
       {
-        "name": "Asher Pembroke"
-      }, 
-      {
-        "affiliation": "University of Texas at Dallas", 
+        "affiliation": "University of Texas at Dallas",
```
Comment (aburrell) [inline .zenodo.json:24]:
```suggestion
        "affiliation": "The University of Texas at Dallas",
```

File: .zenodo.json [hunk 2]
```diff
@@ -1,58 +1,56 @@
...
-    ], 
-    "access_right": "open", 
-    "resource_type": {
-      "type": "software", 
+    ],
+    "access_right": "open",
+    "upload_type": {
+      "type": "software",
       "title": "Software"
     }
```
Comment (aburrell) [inline .zenodo.json]:
```suggestion
```

File: .zenodo.json [hunk 3]
```diff
@@ -1,58 +1,52 @@
...
+        "orcid": "0000-0003-1707-7963"
       }
-    ], 
-    "access_right": "open", 
-    "resource_type": {
-      "type": "software", 
-      "title": "Software"
-    }
+    ],
+    "access_right": "open"
```
Comment (aburrell) [inline .zenodo.json:51]:
```suggestion
    ]
```

**Review by aburrell — APPROVED** [review]

**Comment by rstoneback** [comment]
I also double checked Zenodo.json using the linter linked in the release wiki text. Checks out.

**Pull request merged**